scripts/update_VM_with_exercises.sh

```diff
@@ -0,0 +1,6 @@
+#! /bin/bash
+# script to install in the VM the packages needed for training.
+# Originally introduced for the PSMR2018 training event.
+
+echo "installing ipython notebook and browser"
```

Comment:
message is incorrect. In fact, currently this file is not necessary as it's all elsewhere

Reply:
I will remove this. I was unsure where all this should stay. I initially created it with the idea to call it only if one wants to create a VM with jupyter and the exercises.